app/assets/stylesheets/gradesheet.css.scss

```diff
@@ -296,3 +309,33 @@ input[type="checkbox"]:not(:checked),
 #interaction_badge {
   color: #0882af;
 }
+
+th:not(.sorting_asc) .up-icon {
+  display: none;
+}
+
+th:not(.sorting_desc) .down-icon {
+  display: none;
+}
+
+th:not(.sorting) .sort-icon {
+  display: none;
+}
+
+.sorting_asc .up-icon {
```

Comment:
Are the `inline-block` necessary? `.material-icons` already applies it

![Screenshot 2023-01-14 at 23 20 40](https://user-images.githubusercontent.com/9074856/212522741-715b1628-b7de-495c-8844-949be52597fc.png)

Reply:
Nope, seems like it works fine without it